packages/node/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@counterfactual/node",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "types": "dist/src/index.d.ts",
+  "module": "dist/index.es.js",
+  "iife": "dist/index-iife.js",
```

Comment:
Are all these various export methods actually necessary? We don't actually import this into the Playground environment yet so why don't we hold off on prematurely packaging it up until we need to?